misc/python/materialize/mzcompose/services.py

```diff
@@ -132,6 +132,7 @@ def __init__(
         self.default_storage_size = (
             default_size
             if image
+            and "latest" not in image
```

Comment:
Could also do `":" not in image` in case there are other special tags

Reply:
no, the `:` is present in the image, the tag is `materialized:latest`